bot.py

```diff
@@ -80,10 +117,19 @@ async def on_message(message):
     # Variables
     guild_id = message.guild.id
     channel_id = message.channel.id
-    irs_day = irs_days[guild_id]
+    user_id = message.author.id
     msg = message.content.lower()
-    today = datetime.datetime.now()
-    days = (irs_day - today).days
+
+    ### Bot Cooldown
+    try:
+        if (datetime.datetime.now().timestamp - bot_cooldown[user_id]) < bot_cooldown_seconds:
+            bot_cooldown[user_id] = datetime.datetime.now().timestamp
+            await message.author.send("Woah there, take it easy. Dont spam me please.")
```

Comment:
- [ ] Check that this implementation is correct and works as intended. The intent being to send a DM to the offending user.